src/main/java/com/minecolonies/coremod/colony/buildings/AbstractBuilding.java

```diff
@@ -83,24 +116,25 @@
      * Map to resolve classNameHash to class.
      */
     @NotNull
-    private static final Map<Integer, Class<?>> classNameHashToClassMap = new HashMap<>();
+    private static final Map<Integer, Class<?>>  classNameHashToViewClassMap  = new HashMap<>();
     /*
      * Add all the mappings.
      */
     static
     {
-        addMapping("Baker", BuildingBaker.class, BlockHutBaker.class);
-        addMapping("Blacksmith", BuildingBlacksmith.class, BlockHutBlacksmith.class);
-        addMapping("Builder", BuildingBuilder.class, BlockHutBuilder.class);
-        addMapping("Home", BuildingHome.class, BlockHutCitizen.class);
-        addMapping("Farmer", BuildingFarmer.class, BlockHutFarmer.class);
-        addMapping("Lumberjack", BuildingLumberjack.class, BlockHutLumberjack.class);
-        addMapping("Miner", BuildingMiner.class, BlockHutMiner.class);
-        addMapping("Stonemason", BuildingStonemason.class, BlockHutStonemason.class);
-        addMapping("TownHall", BuildingTownHall.class, BlockHutTownHall.class);
-        addMapping("Warehouse", BuildingDeliveryman.class, BlockHutDeliveryman.class);
-        addMapping("Fisherman", BuildingFisherman.class, BlockHutFisherman.class);
-        addMapping("GuardTower", BuildingGuardTower.class, BlockHutGuardTower.class);
+        addMapping("Baker", BuildingBaker.class, BuildingBaker.View.class, BlockHutBaker.class);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this reference to "BuildingBaker". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2390)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this reference to "BuildingBaker". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2390)